keyboards/ergo42/keymaps/ichi-t/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```

Comment:
This isn't needed.  The `ifndef QUANTUM_DIR` block is a holdover from an older method. 

You can remove the file, or just empty the file

Reply:
I deleted this file.
Thanks.